docs/_sass/_blog.scss

```diff
@@ -32,6 +32,16 @@
     text-decoration: none;
     background-color: darken(#EDF2F6,10);
```

Comment:
funky indentation here

Reply:
Looks weird here in GH, but it's ok in the file. Maybe GH isn't rendering the .scss nesting properly?
<img width="350" alt="screen shot 2019-02-28 at 12 57 29 pm" src="https://user-images.githubusercontent.com/123787/53587505-8c637580-3b58-11e9-92e1-a64f819f89cb.png">